coral/src/services/api-mocks/server.ts

```diff
@@ -0,0 +1,3 @@
+import { setupServer } from "msw/node";
+
+export const server = setupServer();
```

Comment:
Did you prototype if we could add some default handlers here:
```
rest.get('*', throwError),
rest.put('*', throwError),
rest.post('*', throwError),
rest.delete('*', throwError),
```

Reply:
I'm just trying to find some config for that that works and is useful!

Reply:
Problem: We  would need to add the `setupServer()`, server.listen and server.close in the setup after env so it's used in every test suite. Otherwise we can only add a handler to prevent this (still trying to find a good approach) in the test suites we're using mocked apis. Which does not really help to prevent accidental api calls in tests  😓 

Trying to see if I find a way to prevent API calls without having to start the server for every test file.

Reply:
Ok I did find a solution 😬  but I would make a new PR for that, because it needs a little bit refining and is another discussion :D